modules/govuk/manifests/apps/whitehall.pp

```diff
@@ -289,9 +289,21 @@
         try_files $uri @app;
       }
 
+      # Restrict access to the healthcheck page. Grafana needs to be
+      # able to access this from the monitoring machine. All other requests are blocked.
+      # This is only useful for Carrenza, as AWS will not provide static IPs.
+      if $::aws_migration {
```

Comment:
I don't think this will work here as it's in a string designed to go directly in the Nginx configuration. I think you'll need to do something like:

```puppet
hidden_paths          => $::aws_migration ? [$health_check_path] : [],
```

in the `govuk::app::nginx_vhost` section above, and then in here you can have the `location` Nginx block.